### PR TITLE
doc: fix charter typos

### PR DESCRIPTION
**Pull request by knolleary**
Fixes a handful of typos in the charter. No material change to the content.

### CODE REVIEW

**Comment by MylesBorins** [comment]
While these are simply errata changes they still need to be reviewed by the board

/cc @brianwarner @mkdolan

File: CPC-CHARTER.md [hunk 1]
```diff
@@ -295,7 +295,7 @@ from the OpenJS Foundation board.
 **Project**: a technical collaboration effort that is organized through the
 project mentorship process and approved by the TAC.
 
-**Contributors**: contribute code or other artifacts, but do not have the right
+**Contributors**: contribute code or other artefacts, but do not have the right
```
Comment (eemeli) [inline CPC-CHARTER.md]: "artifacts" is valid American English, but do we have a preference for British spelling?

Comment (mhdawson) [reply CPC-CHARTER.md]: A least in the Node.js project the preference is for US English (I think that is even written into the docs).

**Comment by kborchers** [comment]
LGTM - I would also prefer the US English spelling of 'artifacts' though I am also from the US and obviously biased and will not block a merge based on that change. I would ask that we are consistent though and if necessary, document somewhere which version of English we default to for our docs.

File: CPC-CHARTER.md (hunk 1 above)
Comment (knolleary) [reply CPC-CHARTER.md]: Ooops - I thought I had successfully ignored all the US/British alternatives and only fixed true typos. This one slipped past. Will revert this one.

**Comment by knolleary** [comment]
'Artifact' now restored.

**Comment by MylesBorins** [comment]
Approved in board meeting

**Pull request merged**